.werft/util/certs.ts

```diff
@@ -26,6 +26,11 @@ export class InstallCertificateParams {
 }
 
 export async function issueCertificate(werft: Werft, params: IssueCertificateParams, shellOpts: ExecOptions): Promise<boolean> {
+    if (isCertReady(params.certName)){
```

Comment:
Have a separate PR open for those changes: https://github.com/gitpod-io/gitpod/pull/13652